src/lib/model/InboundTransfersModel.js

```diff
@@ -228,9 +283,23 @@ class InboundTransfersModel {
             const mojaloopResponse = {
                 completedTimestamp: new Date(),
                 transferState: 'COMMITTED',
```

Comment:
@kirgene note that this is where the scheme adapter sets the transfer state as committed; the SDK should have no awareness of other states right now. If the call to the DFSP backend fails the scheme adapter returns error, if it succeeds it returns transfer as COMMITTED; there is no middle ground.